test/smoketest_artman.py

```diff
@@ -60,7 +60,6 @@ def run_smoke_test():
 def generate_grpc_library(artman_config, lang):
     grpc_pipeline_args = [
         'artman',
-        '--user-config', 'test/artman_user_config_in_smoketest.yaml',
```

Comment:
This was incompatible with the Docker image; it was working previously by mistake because the local artman directory was being added to the image.